src/components/shared/Validations.tsx

```diff
@@ -3,16 +3,18 @@ import { Dictionary } from '@stoplight/types';
 import { Popover } from '@stoplight/ui-kit';
 import cn from 'classnames';
 import * as React from 'react';
+import { Context as ViewContext } from '../JsonSchemaViewer';
 
 export interface IValidations {
   required: boolean;
-  validations: (Dictionary<unknown> | {}) & { deprecated?: boolean };
+  validations: (Dictionary<unknown> | {}) & { deprecated?: boolean; readOnly?: boolean; writeOnly?: boolean };
```

Comment:
It's a lie, isn't it?
readOnly and writeOnly can be anything. We don't assert the type anywhere, so it's a blind guess (hope).

Reply:
Nope it could be anything. We can make it `unknown` and check if its truthy

Reply:
Yeah, that makes sense.